README.md

```diff
@@ -0,0 +1,76 @@
+# Cognoma core-service
+
+This repository, under the umbrella of Project Cognoma
+(https://github.com/cognoma), holds the source code, under open source
+license, of the Terraform configuration files used to manage the infrastructure
+for the backend of the Project Cognoma.
+
+## Getting started
+
+Make sure to fork [this repository on
+ +GitHub](https://github.com/cognoma/infrastructure "cognoma/infrastructure on
```

Comment:
Oh is that why that plus was there? I copied this from https://github.com/cognoma/core-service/blob/master/README.md#getting-started to have that first step be consistent across cognoma projects. I didn't know why the plus sign was there but I thought it was some weird convention. I'm happy to take it out though.